src/frontend/src/pages/WorkPackageForm/WorkPackageFormView.tsx

```diff
@@ -59,34 +66,43 @@ export interface WorkPackageEditFormPayload {
   }[];
 }
 
-const WorkPackageEditContainer: React.FC<WorkPackageEditContainerProps> = ({ workPackage, exitEditMode }) => {
+const WorkPackageFormView: React.FC<WorkPackageFormViewProps> = ({
+  exitActiveMode,
+  mutateAsync,
+  defaultValues,
+  wbsElement,
+  leadOrManagerOptions,
+  blockedByOptions,
+  crId
+}) => {
   const toast = useToast();
   const user = useCurrentUser();
-  const query = useQuery();
-  const allUsers = useAllUsers();
-  const { name, startDate, duration } = workPackage;
   const {
     register,
     handleSubmit,
     control,
     formState: { errors }
   } = useForm({
     resolver: yupResolver(schema),
-    defaultValues: {
-      workPackageId: workPackage.id,
-      name,
-      crId: query.get('crId') || '',
-      stage: workPackage.stage || 'NONE',
-      startDate,
-      blockedBy: workPackage.blockedBy.map(wbsPipe),
-      duration,
-      expectedActivities: bulletsToObject(workPackage.expectedActivities),
-      deliverables: bulletsToObject(workPackage.deliverables)
-    }
+    defaultValues: defaultValues
+      ? {
+          ...defaultValues,
+          crId: crId ?? defaultValues.crId
+        }
+      : {
+          name: '',
+          workPackageId: 0,
+          startDate: getMonday(new Date()),
+          duration: 0,
+          crId: crId ?? '',
+          blockedBy: [],
+          expectedActivities: [],
+          deliverables: []
+        }
```

Comment:
you can also do this like - name: defaultValues.name ?? '', so you basically use the defaultValue if it exists and then provide the beginning value for the create if not. This allows each default value to be set in one line, might be cleaner, but if it causes any weird issues, dw ab doing it